pkg/kubelet/dockershim/docker_stats_windows.go

```diff
@@ -52,6 +52,14 @@ func (ds *dockerService) getContainerStats(containerID string) (*runtimeapi.Cont
 
 	stats, err := hcsshimContainer.Statistics()
 	if err != nil {
+		if strings.Contains(err.Error(), "0x5") || strings.Contains(err.Error(), "0xc0370105") {
+			// When the container is just started quering for stats causes access errors
+			// This is transient; skip container for now
+			// These hcs errors do not have helpers exposed in public package so need to query for the known codes:
+			// https://github.com/microsoft/hcsshim/blob/master/internal/hcs/errors.go
```

Comment:
Can we file a bug asking them to make these public?

Reply:
https://github.com/microsoft/hcsshim/pull/933